tools/block-generator/print_tps.py

```diff
@@ -0,0 +1,72 @@
+import os
+import argparse
+
+TXN_PER_BLOCK = 5000
+TXN_PER_BLOCK_SM = 100
+TXN_PER_BLOCK_JUMBO = 25000
+
+parser = argparse.ArgumentParser(description="Parse block TPS from reports")
+parser.add_argument(
+    "-d",
+    "--perf-report-dir",
+    required=False,
+    dest="dir",
+    help="report directory created by the block generator",
+)
+
+parser.add_argument(
+    "-c",
+    "--conduit-version",
+    required=False,
+    dest="conduit_version",
+    help="Release version or the commit hash of the Conduit binary used during the performance test",
+)
+
+parser.add_argument(
+    "-s" "--database-description",
+    required=False,
+    dest="database_description",
+    help="A short description of the database state used for the performance test",
+)
+
+args = parser.parse_args()
+
+
+def parse_report(report):
+    data = dict()
+    with open(report) as f:
+        for line in f:
+            tag, value = line.split(":")
+            data[tag] = value if tag == "scenario" else float(value)
+    return data
+
+
+def pretty_print(data):
+    table_header = (
+        f"Scenario,Conduit_Version,{args.database_description}"
+    )
+    print(table_header)
+    for d in data:
+        scenario = d["scenario"].split("config.")[1]
+        scenario_parsed = scenario.split(".yml")[0]
+        txn_per_block = TXN_PER_BLOCK
```

Comment:
I don't think this is an issue, but just noting the fact that I'm about to change this exactness of these numbers going forward. For example, formerly the _small_ scenarios had **exactly** `TXN_PER_BLOCK_SM` (100) but now they may have $100 + \epsilon$ .